language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/common/utils/CommonUtil.java

```diff
@@ -137,6 +137,8 @@
 
     public static final boolean COMPILE_OFFLINE;
 
+    public static final boolean STICKY_BUILD_OPTION = true;
```

Comment:
Do we need to expose this? Since we do not have any requirement to expose this, let's hide it and just pass true to the build options. WDYT?

Reply:
+1

Reply:
Done